src/tbp/monty/frameworks/experiments/data_collection_experiments.py

```diff
@@ -54,7 +54,7 @@ def pass_features_to_motor_system(self, observation, step):
             None
             if self.model.motor_system._policy.action is None
             else (
-                f"{self.model.motor_system._policy.action.agent_id}."
+                f"{str(self.model.motor_system._policy.action.agent_id)}."
```

Comment:
Does this not work without the `str` call? I'd expect that the representation of a newtype is the representation of the underlying type.